### PR TITLE
add f03gl and PLplot

### PR DESCRIPTION
**Pull request by milancurcic**
Submitting 2 graphics libraries to the registry. I couldn't find the PR template for this. Do we have any?

### CODE REVIEW

File: _data/package_index.yml [hunk 1]
```diff
@@ -382,6 +382,21 @@
 
 # --- Graphics ---
 
+- name: f03gl
+  url: http://www-stone.ch.cam.ac.uk/pub/f03gl/index.xhtml
+  description: Fortran 2003 interface to OpenGL
+  categories: graphics interfaces
+  tags: graphics interface opengl
+  license: GNU GPL v3
+  version: none
```
Comment (LKedward) [inline _data/package_index.yml:391]:
```suggestion
```
We should omit version here for non-github repos.
(This is my fault: the interface isn't consistent between github and non-github repos).

**Comment by LKedward** [comment]
> Submitting 2 graphics libraries to the registry. I couldn't find the PR template for this. Do we have any?

No PR template yet

File: _data/package_index.yml (hunk 1 above)
Comment (milancurcic) [reply _data/package_index.yml:391]: Why should we omit it? What's the difference between GitHub and non-GitHub projects with regard to version?

**Comment by milancurcic** [comment]
#build_preview

**Comment by milancurcic** [comment]
Preview doesn't seem to work.

File: _data/package_index.yml (hunk 1 above)
Comment (LKedward) [reply _data/package_index.yml:391]: Unfortunately I wasn't very consistent when I wrote the Jekyll logic for displaying the version badge. I should change this.

**Comment by LKedward** [comment]
> Preview doesn't seem to work.

Working for me:
Screenshot from 2020-05-12 19-41-27

**Comment by milancurcic** [comment]
It does for me as well now.

File: _data/package_index.yml (hunk 1 above)
Comment (LKedward) [reply _data/package_index.yml:391]: @milancurcic 
For github repos:
- omitting the version means use the github-detected version
- `version:none` will hide the version tag

For non-github repos:
- omitting the version will hide the version tag
- `version:none` will create a tag showing 'version:none'

**Comment by milancurcic** [comment]
Regarding versions: f03gl indeed didn't have a version and we don't want to show a version none badge. However, PLplot does (5.15.0, I forgot to include it). Why wouldn't we show version for non-GH projects that have one?

**Comment by LKedward** [comment]
Sorry I didn't mean for all non-github repos, just those without explicit versions

**Comment by certik** [comment]
Here is what we have to go over: https://github.com/fortran-lang/fortran-lang.org/blob/8f74576faef57a6da975195ffda8da85402b1ba9/PACKAGES.md

**Comment by milancurcic** [comment]
@certik I did, feel free to check it as well.

**Comment by LKedward** [comment]
> Here is what we have to go over: https://github.com/fortran-lang/fortran-lang.org/blob/8f74576faef57a6da975195ffda8da85402b1ba9/PACKAGES.md

I'm happy these packages meet all the criteria discussed :+1:

**Comment by certik** [comment]
Have you tried installing the packages and do they work? I haven't checked the "Maturity" section. 

I am hoping that fpm will help with libraries like these, to make them easy to install, part of the wide Fortran ecosystem and also have usage statistic at the registry webpage, and so on.

**Comment by milancurcic** [comment]
> Have you tried installing the packages and do they work? I haven't checked the "Maturity" section.

Yes for f03gl, no for PLplot.

Have you looked at PLplot? This may be among the most mature libraries in the registry (contributions go back to 1992): https://www.openhub.net/p/plplot/contributors/summary

**Comment by milancurcic** [comment]
Thank you both, merging.

**Comment by milancurcic** [comment]
#delete_preview

**Pull request merged**